lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,7 +19,20 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
-
+    if not isinstance(text, str):
+        return None
+    new_text = ''
+    for i in text.lower():
```

Comment:
`i` is a good name for `index`, but it is not an index

Reply:
done